test/unit/core/entities/ds_status_test.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hmi_core/hmi_core.dart';
+
+void main() {
+  group('DsStatus', () {
```

Comment:
add test to verify that class has all necessary names with proper values:
  ok(0),
  obsolete(2),
  timeInvalid(3),
  invalid(10);